geomstats/geometry/riemannian_metric.py

```diff
@@ -1,4 +1,6 @@
-"""Riemannian and pseudo-Riemannian metrics."""
+"""
+Riemannian and pseudo-Riemannian metrics.
```

Comment:
Did you rebase your code recently? I think many of the docstring issues with your PR stem from the fact that we merged the docstrings PR from @cshewmake2 a few days ago ...